content/en/docs/ops/diagnostic-tools/istioctl-analyze/index.md

```diff
@@ -115,3 +115,60 @@ the kind of information you should provide.
 - **What about analysis that goes beyond configuration?**
 
       Today, the analysis is purely based on Kubernetes configuration, but in the future we’d like to expand beyond that. For example, we could allow analyzers to also look at logs to generate recommendations.
+
+- **Where can I find out how to fix the errors I'm getting?**
+
+      Reference for configuration analysis messages can be found [here](/docs/reference/config/analysis/).
+
+## Enabling Validation Messages for Resource Status
+
+*Note that this feature is experimental.*
+
+Starting with Istio 1.4, Galley can be set up to perform configuration analysis alongside the configuration distribution that it is primarily responsible for. Validation messages from the analysis are written to the status subresource of the affected Istio resource. This lets you find many configuration errors in your mesh without needing to run a separate command. This analysis uses the same logic and error messages as when using `istioctl analyze`.
```

Comment:
We need to emphasize that this is completely different from 'istioctl analyze'. Specifically:

- You can run analyze against much older clusters, while this only helps with absolute newest clusters
- It only lets you get status messages one by one, so it's not at all a replacement for `analyze`, which gives you everything. In this sense, I find the statement *This lets you find many configuration errors in your mesh without needing to run a separate command* rather confusing.

Is this on by default in 1.4?

Reply:
It is off by default in 1.4.

Added a section with some caveats / explanations of what's different.